test/local/routes/sms.js

```diff
@@ -589,3 +589,103 @@ describe('/sms/status with disabled geo-ip lookup', () => {
   })
 })
 
+describe('/sms/status with query param and enabled geo-ip lookup', () => {
```

Comment:
FWIW, if I allow any country on the front end and hit the backend with a bunk country, it behaves as expected.